brownie/test/strategies.py

```diff
@@ -158,6 +159,15 @@ def strategy(type_str: str, **kwargs: Any) -> SearchStrategy:
     if type_str == "fixed168x10":
         return _decimal_strategy(**kwargs)
     if type_str == "address":
+        if "exclude" in kwargs:
+            if not network.is_connected():
```

Comment:
This will trigger a connection/disconnection for every test that uses `exclude`, which adds about 3 seconds per test on my machine.  It could also have issues if ganache isn't using deterministic addresses for the local accounts.

I think it would be better to modify the [deferred strategy](https://hypothesis.readthedocs.io/en/latest/data.html#hypothesis.strategies.deferred) logic on line 82 and apply the exclusion when the lambda function executes to create the data set. This approach wouldn't add the overhead of connecting/disconnecting.

Reply:
I had originally tried to put there but I wasn't able to get it working. I think that's because I was using it wrong. I'll give it another try and see if I can get it working there.

Reply:
@iamdefinitelyahuman did I complete these changes satisfactorily or is there more I need to do?